.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 repos:
 - repo: https://github.com/ambv/black
-  rev: 19.10b0
+  rev: 21.12b0
```

Comment:
This seems to be different than the version pinned in `pyproject.toml`

Reply:
good catch